GitCommands/CommitDataManager.cs

```diff
@@ -206,6 +179,25 @@ public CommitData CreateFromRevision(GitRevision revision)
                 revision.Body ?? revision.Subject);
         }
 
+        [ContractAnnotation("=>false,error:notnull,data:null")]
+        [ContractAnnotation("=>true,error:null,data:notnull")]
+        private bool TryGetCommitLog([NotNull] string commitId, [NotNull] string format, out string error, out string data)
```

Comment:
nit: I would move it after `GetModule()` to help with A-Z order and over maintainability

Reply:
Sure, I'll move it.

My rationale for that positioning: It's a private method only used by the two methods immediately before it. They're logically related to one another. `GetModule` is in turn used by that, so introduced later still. When there are only a few members to a type, I tend to order them top to bottom. I've seen other codebases (three.js comes to mind) where members are ordered bottom to top, so as you read down the class you see the definitions before they're used.